h2o-core/src/main/java/water/api/schemas3/ModelOutputSchemaV3.java

```diff
@@ -19,6 +19,9 @@
   @API(help="Column names", direction=API.Direction.OUTPUT)
   public String[] names;
 
+  @API(help="Column types", direction=API.Direction.OUTPUT)
+  public String[] column_types;
```

Comment:
The output of this should be a finite set, basically defined by `Vec.T_*` constants. Which means it should be documented somewhere which values our users might expect.